man/systemd.link.xml

```diff
@@ -946,6 +946,16 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MDI-X=</varname></term>
+        <listitem>
+          <para>Specifies MDI-X mode for the port. Takes a boolean value or special value
```

Comment:
some brief explanation what MDI-X actually is would be great. After reading this man page I am now more confused than before.